### PR TITLE
docs: change default format to Parquet.

### PR DESCRIPTION
**Pull request by youngsofun**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

doc update about https://github.com/datafuselabs/databend/pull/9956
keep it a draft before   https://github.com/datafuselabs/databend/pull/9956 is merged

### CODE REVIEW

File: docs/doc/12-load-data/00-stage.md [hunk 1]
```diff
@@ -180,7 +180,7 @@ COPY INTO books FROM @my_int_stage files=('books.csv') file_format = (type = CSV
 <TabItem value="parquet" label="Parquet">
 
 ```sql
-COPY INTO books FROM @my_int_stage files=('books.parquet') file_format = (type = 'Parquet');
+COPY INTO books FROM @my_int_stage files=('books.parquet') file_format = (type = Parquet);
```
Comment (BohuTANG) [inline docs/doc/12-load-data/00-stage.md:183]: `Parquet` -> `PARQUET`

File: docs/doc/13-sql-reference/75-file-format-options.md [hunk 2]
```diff
@@ -14,6 +14,8 @@ FILE_FORMAT = ( TYPE = { CSV | TSV | NDJSON | PARQUET | XML } [ formatTypeOption
 Databend currently supports XML as a source ONLY. Unloading data into an XML file is not supported yet.
 :::
 
+If `FILE_FORMAT` is not specified, use `FILE_FORMAT = (type = Parquet)` by default.
```
Comment (BohuTANG) [inline docs/doc/13-sql-reference/75-file-format-options.md:17]: `Parquet` -> `PARQUET`

File: docs/doc/12-load-data/00-stage.md (hunk 1 above)
Comment (youngsofun) [reply docs/doc/12-load-data/00-stage.md]: fixed

File: docs/doc/13-sql-reference/75-file-format-options.md (hunk 2 above)
Comment (youngsofun) [reply docs/doc/13-sql-reference/75-file-format-options.md]: done

**Review by BohuTANG — APPROVED** [review]

**Pull request merged**